packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -0,0 +1,170 @@
+
+import requests
+
+from .types import *
+from .exceptions import CorelliumApiException
+
+
+class ApiClient:
+    """
+    Corellium ReST API client used by the Corellium driver.
+    """
+    session: Session
```

Ruff (0.8.2)

`Session` may be undefined, or defined from star imports

(F405)